its/org.sonarlint.eclipse.its/src/org/sonarlint/eclipse/its/RulesConfigurationTest.java

```diff
@@ -164,10 +202,34 @@ public String getFailureMessage() {
           .collect(Collectors.joining(","));
       }
     });
-    
+
     bot.treeWithId("slRuleTree")
       .getTreeItem("Java").getNode(0)
       .select().click();
   }
 
+  void changeRuleParamValue(IProject project) throws CoreException {
+    openComplexityRule();
+    bot.spinner().setSelection(10);
+    bot.button("Apply").click();
+    bot.activeShell().close();
+    checkIssueChanged(project);
+  }
+
+  static void checkIssueChanged(IProject project) throws CoreException {
```

Comment:
maybe rename `checkXXX` with `assertXXX` for consistency with assertj assertions?